spynnaker/pyNN/external_devices_models/external_device_lif_control.py

```diff
@@ -20,12 +20,13 @@
 from spynnaker.pyNN.models.defaults import default_initial_values,\
     default_parameters
 from spynnaker.pyNN.models.neuron.input_types import InputTypeCurrent
-from spynnaker.pyNN.models.neuron.neuron_models import (
-    NeuronModelLeakyIntegrateAndFire)
 from spynnaker.pyNN.models.neuron.synapse_types import SynapseTypeExponential
 from .external_device_lif_control_vertex import ExternalDeviceLifControlVertex
 from .threshold_type_multicast_device_control import (
     ThresholdTypeMulticastDeviceControl)
+from ..models.neuron.neuron_models.\
```

Comment:
1) Don't use relative imports like this.  Only import from the current folder, or else go for the full thing.
2) Why do you need this?  You are using standard LIF in the C code...

Reply:
coz i couldnt get the compiler to play ball without it.

Reply:
The python compiler?  You  should be able to use just the LIF model I believe

Reply:
1. was pycharm doing pycharmness. am correcting when replacing with old lif

Reply:
fixed